Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -190,6 +193,55 @@ void ProjectData::parseProcessVariables(
 	}
 }
 
+void ProjectData::parseParameters(ConfigTree const& parameters_config)
+{
+	using namespace ProcessLib;
+
+	DBUG("Reading parameters:");
+	for (auto pc_it : parameters_config)
+	{
+		// Skip non-parameter section.
+		if (pc_it.first != "parameter")
+			continue;
+		ConfigTree const& parameter_config = pc_it.second;
+
+		auto name = parameter_config.get_optional<std::string>("name");
+		if (!name)
+		{
+			ERR("The parameter config does not provide a name tag.");
+			std::abort();
+		}
+
+		auto type = parameter_config.get_optional<std::string>("type");
+		if (!type)
+		{
+			ERR("Could not find required parameter type.");
+			std::abort();
+		}
+
+		// Create parameter based on the provided type.
+		if (*type == "Constant")
+		{
+			INFO("ConstantParameter: %s.", name->c_str());
+			_parameters.push_back(createConstParameter(parameter_config));
+			_parameters.back()->name = *name;
+		}
+		else if (*type == "MeshProperty")
+		{
+			INFO("MeshPropertyParameter: %s", name->c_str());
+			_parameters.push_back(
+			    createMeshPropertyParameter(parameter_config, *_mesh_vec[0]));
```

Comment:
Is it correct that properties are only assignable to the first mesh in `_mesh_vec`?